src/com/trainee/aizaz/java/AscendingOrderArray.java

```diff
@@ -0,0 +1,49 @@
+package com.trainee.aizaz.java;
+import java.util.InputMismatchException;
+import java.util.Scanner;
+/*
+* The purpose of this program is to sort the elemnts in ascending order
+* without predefine functions
+* */
+
+public class AscendingOrderArray {
+    public static void main(String[] args) {
+        Scanner sc = new Scanner(System.in);
+        System.out.println("Enter the size of the array:");
+        try {
+            int n = sc.nextInt();
+            int[] arr = new int[n];
+            //insterion of array elements:
+            for (int i = 0; i < arr.length; i++) {
+                System.out.println("Enter arr[" + i + "]=");
+                arr[i] = sc.nextInt();
+            }
+            //sorting of elements
+            for (int i = 0; i < arr.length; i++) {
+                for (int j = 0; j < arr.length; j++) {
+                   //Logic for sorting in ascending order
+                    if (arr[i] < arr[j]) {
```

Comment:
Comment should go inline with if 
//Logic for sorting in ascending order
if (arr[i] < arr[j]) {